pvlib/pvsystem.py

```diff
@@ -1598,10 +1594,21 @@ def get_cell_temperature(self, poa_global, temp_air, wind_speed, model,
                 'module_height', 'wind_height', 'emissivity', 'absorption',
                 'surface_tilt', 'module_width', 'module_length'],
                 self.temperature_model_parameters)
-            # default to using the Array attribute, but allow user to override
-            # with a custom surface_tilt value in temperature_model_parameters
+            # use surface_tilt from temp_model_params if available, otherwise
+            # fall back to the mount value.
+            # TODO: this logic is probably not what we want.
             if 'surface_tilt' not in optional:
-                optional['surface_tilt'] = self.surface_tilt
+                # should this use mount.get_orientation?  we don't have
+                # solar position available here...
+                try:
+                    optional['surface_tilt'] = self.mount.surface_tilt
+                except AttributeError:
+                    msg = (
+                        "surface_tilt is required for the fuentes model; "
+                        "specify it as a key in temperature_model_parameters "
+                        "or as an attribute of the Array's Mount."
+                    )
+                    raise ValueError(msg)
```

Comment:
I'm not sure what to do about the various places to look for `surface_tilt` now that the `PVSystem` and its `Mount` aren't guaranteed to have a `surface_tilt` attribute.  Is it reasonable to remove this complexity and only look in `temperature_model_parameters`?

Reply:
>  Is it reasonable to remove this complexity and only look in temperature_model_parameters?

Sounds good to me. 

Probably a bad idea: An alternative would be calling `get_orientation` with something like `0, 0` - doesn't matter for fixed tilt systems and it would make most single axis trackers return 0.

Definitely not interested in adding solar position kwargs to this method.